components/sup/src/manager/service/dir.rs

```diff
@@ -116,6 +116,12 @@ impl<'a> SvcDir<'a> {
     where
         P: AsRef<Path>,
     {
+        // We do not want to change the permissions of an already
+        // existing directory (on linux)
```

Comment:
This comment implies that something else would be done on Windows, but the code doesn't reflect that. 

Is the code right, or is the comment?

Reply:
window's doesn't do anything with permissions. I can get rid of it if it's confusing

Reply:
Yeah, let's go ahead and remove the "(on linux)" bit. Thanks!